lib/phoenix/router.ex

```diff
@@ -210,15 +210,15 @@ defmodule Phoenix.Router do
 
   Scopes allow us to scope on any path or even on the helper name:
 
-      scope "/v1", MyAppWeb, host: "api." do
+      scope "/api/v1", MyAppWeb, as: :api_v1 do
         get "/pages/:id", PageController, :show
       end
 
   For example, the route above will match on the path `"/api/v1/pages/1"`
-  and the named route will be `api_v1_page_path`, as expected from the
+  and the named helper will be `api_v1_page_path`, as expected from the
```

Comment:
I updated this to "named helper" for consistency with the rest of the document, where the term is used. "named route" is not used here, and I didn't find info about it via google, so I believe it's a typo